reports/report-reactionlistener.go

```diff
@@ -68,27 +62,39 @@
                         {{$action := ""}}
                         {{if eq .Reaction.Emoji.Name "⚠"}}
                             {{if (reFind `(?i)ManageMessages` (exec "viewperms 204255221017214977"))}}
+                                {{sendMessage nil "30s. Type out the reason why you want to warn that user."}}
+				                {{sleep 30}}
```

Comment:
erm... sleep 30s sounds hellish.. sleep stops the gourotines (ye idk how to spell it) so it essentially stops the entire cmd for 30s... maby an execCC loop similar to ;delay in yaggy support